src/Console/Command/ReleaseCommand.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DevKit\Console\Command;
+
+use Github\Exception\ExceptionInterface;
+use Packagist\Api\Result\Package;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Jordi Sala <jordism91@gmail.com>
+ */
+final class ReleaseCommand extends AbstractCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this
+            ->setName('release')
+            ->setDescription('Makes a release of a project.')
+            ->addArgument('project', InputArgument::REQUIRED, 'The project to do a release of.')
+            ->addOption('branch', null, InputOption::VALUE_OPTIONAL, 'Base branch to release', '3.x')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $project = $input->getArgument('project');
+        $branch = $input->getOption('branch');
+
+        try {
+            $package = $this->packagistClient->get(static::PACKAGIST_GROUP.'/'.$project);
+            $this->io->title($package->getName());
+            $this->prepareRelease($package, $branch, $output);
+        } catch (ExceptionInterface $e) {
+            $this->io->error('Failed with message: '.$e->getMessage());
```

Comment:
What's the point? Also, maybe not return 0?

Reply:
Let's remove the try catch, it just makes things like bad credentials harder to troubleshoot IMO